public/pages/Findings/containers/Findings/Findings.tsx

```diff
@@ -213,6 +217,20 @@ export default class Findings extends Component<FindingsProps, FindingsState> {
     this.setState({ notificationChannels: channels });
   };
 
+  async getPlugins() {
```

Comment:
Same as above, let's make this a shared utility function

Reply:
done above